src/generate.ts

```diff
@@ -32,7 +33,11 @@ export function generate(src: string = conf.apiFile, dest: string = conf.outDir)
   }
 
   const header = processHeader(schema);
-  const config: Config = {header, dest};
+  const config: Config = {
+    header,
+    dest,
+    baseUrl: `${schema.schemes[0] || 'http'}://${schema.host}${schema.basePath}`,
```

Comment:
`schemes` is an optional attribute according to the [spec](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).

Reply:
That's why I use  `http` as default value

Reply:
Not really, evaluate `const o = {}; o.s[0] || 'h'` in the console.

Reply:
Fixed